src/GafferBindings/SplinePlugBinding.cpp

```diff
@@ -54,6 +55,89 @@ using namespace Gaffer;
 namespace
 {
 
+template<typename T>
+std::string splineDefinition_repr( T &x )
+{
+	return "";
+}
+
+
+#define REPR_SPLINEDEFINITION_SPECIALISATION( TYPE )                                                    \
```

Comment:
Looks like this approach is taken from the Cortex spline bindings? If you're up for it, I think we can do a bit better than resorting to macros to replace a default implementation that silently fails :

```
template<typename T>
std::string splineDefinitionRepr( object x )
{
    string stream s;
    /// Query name from python object itself, so we don't need the macro.
    s << "Gaffer." << extract<std::string>( object.attr( "__name__" ) ) << "( ";
    /// Use extract to get the spline ready for extracting the points etc.
    const T splineDefinition = extract<T>( x );
    ...
}
```